pkg/api/speak-stream/v1/router.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2023-2024 Deepgram SDK contributors. All Rights Reserved.
+// Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+// SPDX-License-Identifier: MIT
+package speak
+
+import (
+	"encoding/json"
+	"os"
+	"strings"
+
+	prettyjson "github.com/hokaccha/go-prettyjson"
+	klog "k8s.io/klog/v2"
+
+	interfaces "github.com/deepgram/deepgram-go-sdk/pkg/api/speak-stream/v1/interfaces"
+)
+
+// MessageRouter routes events
+type MessageRouter struct {
+	callback       interfaces.SpeakMessageCallback
+	debugWebsocket bool
+}
+
+// NewWithDefault creates a MessageRouter with the default callback handler
+func NewWithDefault() *MessageRouter {
+	return NewStream(NewDefaultCallbackHandler())
+}
+
+// New creates a MessageRouter with a user-defined callback
+func NewStream(callback interfaces.SpeakMessageCallback) *MessageRouter {
+	debugStr := os.Getenv("DEEPGRAM_DEBUG_WEBSOCKET")
```

Comment:
Can we revert this to `DEEPGRAM_DEBUG`? We don't need debugging differentiation between REST and WS. You are most likely going to use one of the other and if you do happen to use both, you probably want to see the debug for both.